src/algorithms/highlevel/sbic.h

```diff
@@ -54,7 +56,7 @@ class SBic : public Algorithm {
     declareParameter("size2", "second pass window size [frames]", "[1,inf)", 200);
     declareParameter("inc2", "second pass increment [frames]", "[1,inf)", 20);
     declareParameter("cpw", "complexity penalty weight", "[0,inf)", 1.5);
-    declareParameter("minLength", "minimum length of a segment [frames]", "[1,inf)", 10);
+    declareParameter("minLength", "minimum length of a segment [frames]. EDIT LANDR 2016: Parameter disabled!", "[1,inf)", 10);
```

Comment:
It means that whatever value you put for that parameter will be ignored. I could also remove the parameter entirely, if that would be preferable.